src/views/DetailView.js

```diff
@@ -0,0 +1,23 @@
+import { VivViewerLayer, StaticImageLayer } from '../layers';
+import VivView from './VivView';
+import { getVivId } from './utils';
+
+/**
+ * This class generates a VivViewerLayer and a view for use in the VivViewer as a detailed view.
+ * */
+export default class DetailView extends VivView {
+  getLayers({ props }) {
+    const { loader } = props;
+    const { id } = this;
+    const layer = loader.isPyramid
+      ? new VivViewerLayer(props, {
+          id: loader.type + getVivId(id),
```

Comment:
Usually I prefer template literals for this kind of thing  (\`${loader.type}${getVivId(id)}\`) simply because it's clearer to me that result is a string.

Reply:
Ok.